src/appliance-onboarding-script/appliance_setup/pkgs/_appliance_setup.py

```diff
@@ -203,14 +207,19 @@ def _deploy_and_create_appliance(self) -> str:
         with TempChangeDir(self._temp_dir):
             config = self._config
             apiserver_address = config['applianceControlPlaneIpAddress']
-            if is_api_server_reachable and not confirm_prompt(f'An ApiServer is already reachable on endpoint {apiserver_address}. Deployment will be skipped. Do you want to continue?'):
+            # Removing confirm_prompts for automated testing 
+            # TODO Check what needs to be done in case there is a reachable ApiServer
+            # For now, we consider the Api server to be correctly configured and don't deploy a new one.
+            if is_api_server_reachable and (self._isAutomated or not confirm_prompt(f'An ApiServer is already reachable on endpoint {apiserver_address}. Deployment will be skipped. Do you want to continue?')):
```

Comment:
if isAutomated is true, do we want to exit or just continue in this case? I think this current code will exit. Is that what we want?